migrations/versions/888b56ed5dcb_.py

```diff
@@ -0,0 +1,40 @@
+"""v3.6: Add table for custom user attributes
+
+Revision ID: 888b56ed5dcb
+Revises: d5870fd2f2a4
+Create Date: 2021-02-10 12:17:40.880224
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '888b56ed5dcb'
+down_revision = 'd5870fd2f2a4'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    try:
+        op.create_table('customuserattribute',
+            sa.Column('id', sa.Integer(), nullable=False),
+            sa.Column('user_id', sa.Unicode(length=320), nullable=True),
+            sa.Column('resolver', sa.Unicode(length=120), nullable=True),
+            sa.Column('realm_id', sa.Integer(), nullable=True),
+            sa.Column('Key', sa.Unicode(length=255), nullable=False),
+            sa.Column('Value', sa.UnicodeText(), nullable=True),
+            sa.Column('Type', sa.Unicode(length=100), nullable=True),
+            sa.ForeignKeyConstraint(['realm_id'], ['realm.id'], ),
+            sa.PrimaryKeyConstraint('id')
+        )
+        op.create_index(op.f('ix_customuserattribute_resolver'), 'customuserattribute', ['resolver'], unique=False)
+        op.create_index(op.f('ix_customuserattribute_user_id'), 'customuserattribute', ['user_id'], unique=False)
+    except Exception as exx:
+        print("Could not add table 'userattribute'.")
```

Comment:
It was important to me to point out, that it is "custom". Also in the code we are speaking of "custom" attributes, since, "user attributes" are sometimes used as the attributes, that are contained in the user store. And the ORM actually introduces the name of the table in the code. And then I would have a "UserAttribute" in the code, which sounds too similar to "UserInfo".

So I ended up with this very long name.
I would like to keep the word "custom".